jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -0,0 +1,629 @@
+import pytest
+import numpy as np
+
+from jwst.ramp_fitting.ramp_fit import ramp_fit
+from jwst.datamodels import dqflags
+from jwst.datamodels import MIRIRampModel
+from jwst.datamodels import GainModel, ReadnoiseModel
+
+
+#@pytest.mark.skip(reason="not using now")
+def test_nocrs_noflux():
+        #all pixel values are zero. So slope should be zero
+    model1, gdq, rnModel, pixdq, err, gain = setup_inputs(ngroups=5)
+    slopes = ramp_fit(model1, 60000, False, rnModel, gain, 'OLS', 'optimal')
+    print(slopes[0].data)
```

Comment:
I would suggest removing the `print()` statements in these tests, as they are not necessary for the tests to run as CI unit tests.  It looks like they were there for debugging purposes in setting up your `assert` statements?  I think they can be removed now.

Reply:
I have removed the print and pytest.mark.skip lines.